plugins/hotkeyshortcuts.koplugin/defaults.lua

```diff
@@ -0,0 +1,123 @@
+-- typed on my Commodore 64 ;)
+local Device = require("device")
+
+-- The hotkey shortcuts defined here are only the defaults. The user can
+-- change them at any time using the hotkey shortcuts configuration menu.
+-- NOTE: The combinations that contain the tag ** keep nil, already assigned outside this plugin **
+--       should never be assigned to any action in this plugin, because they are already assigned
+--       to events in core. If you assign them to actions in this plugin, those actions will be in
+--       conflict with the existing ones and hell will break loose.
```

Comment:
_should never_
Then, why mention them ? Why not this if you want to keep the symetry ?:
```lua
        modifier_plus_up                 = nil,
   --   modifier_plus_down               -- not assignable
        modifier_plus_left               = nil,
```

Reply:
Honestly, because it never occurred to me ;).